src/owasp-dependency-check-suppressions.xml

```diff
@@ -375,65 +392,29 @@
     <!-- debezium-related misdetections -->
     <suppress>
         <notes><![CDATA[
-       file name: debezium-connector-mysql-1.9.5.Final.jar
+       file name: debezium-connector-mysql-1.9.7.Final.jar
```

Comment:
These debezium-related misdetections are exactly included with previous suppressions. I don't know which PR bump the version so that the suppression becomes invalid.

Reply:
It's https://github.com/apache/pulsar/pull/18465 authored by @dlg99. But we skip the OWASP check in the PR.